tools/azure-pipelines/build-apache-repo.yml

```diff
@@ -126,3 +127,8 @@ stages:
             versionSpec: '= 2.4'
             addToPath: true
         - script: ./tools/travis/docs.sh
+      - template: build-python-wheels.yml
+        parameters:
+          stage_name: build_python_wheels
```

Comment:
```suggestion
          stage_name: cron_python_wheels
```

Reply:
This is more in line with the other `stage_name`s: `cron_hadoopfree`, `cron_scala212`, `cron_jdk11`, `cron_hadoop241`.